rustls-platform-verifier/src/verification/others.rs

```diff
@@ -54,11 +54,17 @@ impl Verifier {
     /// WebPKI, using root certificates provided by the platform and augmented by
     /// the provided extra root certificates.
     pub fn new_with_extra_roots(
-        roots: impl IntoIterator<Item = pki_types::TrustAnchor<'static>>,
+        roots: impl IntoIterator<Item = impl Into<pki_types::CertificateDer<'static>>>,
```

Comment:
As for macOS, I prefer skipping the `Into` indrection here.

Also noteworthy that this breaks compatibility. @complexspaces do you plan on backporting this stuff to 0.1?

Reply:
> (Maybe tack on a commit that bumps the version number to 0.4?)

Thanks for the suggestion, that's done

Reply:
We should consider adding cargo-semver-check to this repo's CI. It looks like we haven't done that yet. It's not perfect, but pretty good at flagging this sort of change.

Reply:
+1 for adding `cargo-semver-check` here. It runs pretty fast in CI so it shouldn't have any downsides even if it doesn't catch everything.

> . @complexspaces do you plan on backporting this stuff to 0.1?

I'm not sure if that's possible under the same SemVer release series. AFAIK 1P has no need for extra roots on the other platforms so I don't think its worth trying to figure out a workaround for either. Does that make sense?

Reply:
Yeah, was wondering if there was a need at 1P.